modules/serverless-kafka-sink.adoc

```diff
@@ -22,7 +22,7 @@ You can create an event sink called a Kafka sink that sends events to a Kafka to
 .Kafka sink YAML
 [source,yaml]
 ----
-apiVersion: eventing.knative.dev/v1alpha1
+apiVersion: eventing.knative.dev/v1beta1
```

Comment:
Only the operator ones with prefix  ```operator.serverless.openshift.io/``` for Serving & Eventing have changed. cc @pierDipi